From 81dc0556bf56b96e92315cfad821295dee4dbade Mon Sep 17 00:00:00 2001
From: pugaizai <i@pugai.life>
Date: Wed, 18 Sep 2024 02:15:10 +0800
Subject: [PATCH] aisub

---
 app/ai_sub/recognizer.lua | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/app/ai_sub/recognizer.lua b/app/ai_sub/recognizer.lua
index 5e5871b..17af798 100644
--- a/app/ai_sub/recognizer.lua
+++ b/app/ai_sub/recognizer.lua
@@ -1,7 +1,7 @@
 sub_recognizer = {
-    _whisper_path = env.app_path .. '/whisper/main.exe',
-    _cuda_whisper_path = env.app_path .. '/whisper/cuda/main.exe',
-    _ffmpeg_path = env.app_path .. '/../../../ffmpeg.exe',
+    _whisper_path = env.app_path .. '/../../../whisper',
+    _cuda_whisper_path = env.app_path .. '/../../../whisper-cuda',
+    _ffmpeg_path = env.app_path .. '/../../../ffmpeg',
     _tmp_counter = 0,
     _cur_msg_callback = nil,
 }
-- 
2.43.0

