From 2e9e97c2d94be902c3c7295919ee1fd82d88438e Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=E9=93=BA=E7=9B=96=E5=B4=BD?= <i@pugai.life>
Date: Mon, 8 Sep 2025 00:46:12 +0800
Subject: [PATCH] disable-dandanapi

---
 KikoPlay.pro | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/KikoPlay.pro b/KikoPlay.pro
index 153bff8..9e21b12 100644
--- a/KikoPlay.pro
+++ b/KikoPlay.pro
@@ -23,7 +23,7 @@ DEFINES += QT_MESSAGELOGCONTEXT
 # deprecated API in order to know how to port your code away from it.
 DEFINES += QT_DEPRECATED_WARNINGS
 DEFINES += ZLIB_WINAPI
-DEFINES += KSERVICE
+
 # You can also make your code fail to compile if you use deprecated APIs.
 # In order to do so, uncomment the following line.
 # You can also select to disable deprecated APIs only up to a certain version of Qt.
