From 4120f793c8c757243e9a2c4b8c80ad56dbc74180 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=E9=93=BA=E7=9B=96=E5=B4=BD?= <i@pugai.life>
Date: Mon, 8 Sep 2025 01:39:30 +0800
Subject: [PATCH] fix build

---
 LANServer/dlna/dlnamediaitem.cpp | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/LANServer/dlna/dlnamediaitem.cpp b/LANServer/dlna/dlnamediaitem.cpp
index e9d91d8..178001b 100644
--- a/LANServer/dlna/dlnamediaitem.cpp
+++ b/LANServer/dlna/dlnamediaitem.cpp
@@ -2,6 +2,9 @@
 #include <QXmlStreamWriter>
 #include <QMimeDatabase>
 #include <QCryptographicHash>
+#include <QMap>
+#include <QHash>
+#include <QString>
 
 namespace
 {
